applications/tari_dan_wallet_daemon/src/handlers/nfts.rs

```diff
@@ -37,6 +45,56 @@ use crate::{
 
 const LOG_TARGET: &str = "tari::dan::wallet_daemon::handlers::nfts";
 
+pub async fn handle_get_nft(
+    context: &HandlerContext,
+    token: Option<String>,
+    req: GetAccountNftRequest,
+) -> Result<GetAccountNftResponse, anyhow::Error> {
+    let sdk = context.wallet_sdk();
+    sdk.jwt_api().check_auth(token.clone(), &[JrpcPermission::Admin])?;
+
+    let non_fungible_api = sdk.non_fungible_api();
+
+    let non_fungible = non_fungible_api
+        .non_fungible_token_get_by_nft_id(req.nft_id)
+        .map_err(|e| anyhow!("Failed to get non fungible token, with error: {}", e))?;
+    let token_symbol = non_fungible.token_symbol.clone();
+    let is_burned = non_fungible.is_burned;
+    let metadata = non_fungible.metadata.into_json();
```

Comment:
To avoid adding serde_json as a dependency to every wasm 
```suggestion
    let metadata = serde_json::to_value(&non_fungible.metadata)?;
```